k8sgpt-operator/uninstall.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+NS=k8sgpt-operator-system
+
+kubectl config set-context --current --namespace=$NS
+
+# Uninstall chart
+helm uninstall k8sgpt-operator
```

Comment:
this would need to be namespaces for the uninstall, might be bit of a moot point since the next step is deleting the namespace but figured I'ed mention for the helm operation to pass.
```suggestion
helm uninstall k8sgpt-operator -n k8sgpt-operator-system
```